cob_bringup/drivers/usb_camera_node.launch

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+
+<launch>
+  <arg name="camera_name" default="head_cam"/>
+  
+  <node name="$(arg camera_name)" pkg="usb_cam" type="usb_cam_node" output="screen" >
+    <param name="video_device" value="/dev/video0" />
+    <param name="image_width" value="2048" />
+    <param name="image_height" value="1536" />
+
+    <param name="pixel_format" value="mjpeg" />
+    <param name="camera_frame_id" value="usb_cam" />
```

Comment:
we should use ```"$(arg camera_name)_link"```, same as in https://github.com/ipa320/cob_robots/blob/indigo_dev/cob_bringup/drivers/openni2.launch